detect-common-errors.sh

```diff
@@ -7,13 +7,17 @@ pass=0
 fail=0
 
 function prepare_venv() {
-    VIRTUALENV=$(which virtualenv)
+    VIRTUALENV="$(which virtualenv)"
     if [ $? -eq 1 ]; then
-        # python34 which is in CentOS does not have virtualenv binary
-        VIRTUALENV=$(which virtualenv-3)
+        # python36 which is in CentOS does not have virtualenv binary
+        VIRTUALENV="$(which virtualenv-3)"
+    fi
```

Comment:
@yzainee virtual env doesnt work with actions it uses python -m venv just modified the script a lil bit so cent os as well as actions can use it

Reply:
ok. just be sure that we dont run e2e for graph cve sync. JFYI

Reply:
got it okay